src/services/millions-draw.service.ts

```diff
@@ -13,14 +14,26 @@ export class MillionsDrawService {
         return this._repository;
     }
 
-    getById = async (id: string): Promise<MillionsDrawEntity> => {
+    getById = async (id: string): Promise<MillionsDrawEntity | null> => {
         return this._repository.findOne({ where: { id } });
     };
 
     exist = async (id: string): Promise<boolean> => {
         return this._repository.exist({ where: { id } });
     };
 
+    existMoreThan = async (id: string, timeInSec: number): Promise<boolean> => {
+        const date = dayjs();
+        date.subtract(timeInSec, 'second');
```

Comment:
is timeInSec something that could change over time ?

Otherwise, you could use "week" from subtract method:
`date.subtract(timeInWeek, "week")` which ease the calculation computation in sec when passing the argument to `existMoreThan` method

https://day.js.org/docs/en/manipulate/add#list-of-all-available-units